### PR TITLE
Update auth.py

### PR DESCRIPTION
**Pull request by Jawschamp**
Slight rewording/typo fix about accidently leaking your auth data, not really worthy of a new update lol, just added to a planned release.

### CODE REVIEW

**Comment by xMistt** [comment]
I'd prefer if you kept `everything should be fine.` since changing password doesn't update existing device auths to new credentials, it just invalidates old ones.

**Comment by Jawschamp** [comment]
> I'd prefer if you kept `everything should be fine.` since changing password doesn't update existing device auths to new credentials, it just invalidates old ones.

I agree to a point, however leaving "everything should be fine" isn't the best thing you wanna read, especially for more sophisticated applications. Up to you, maybe rewording it to say something like "resetting your password will not update anything, it just invalidates past device auth data." Basically exactly what you said. Hope that sounds better.

**Comment by xMistt** [comment]
yeah i like this more than the original thanks
